samples/fpgaperf_counter/fpgaperf_counter.h

```diff
@@ -65,6 +65,7 @@ typedef struct {
 typedef struct {
 	pthread_mutex_t lock;
 	uint64_t magic;
+	bool previlege;
```

Comment:
Then, this flag can be removed.  Add the privilege check in host_exerciser_cmd.h:174, just before the perf counters are queried.